tests/acceptance/features/webUIResharing1/reshareUsers.feature

```diff
@@ -183,17 +183,6 @@ Feature: Resharing shared files with different permissions
       | item_type   | folder                |
       | permissions | delete, read          |
 
-  @issue-5461
-  Scenario: User is not allowed to reshare a file/folder with the higher permissions
-    Given user "Brian" has shared folder "simple-folder" with user "Alice" with "read, share, delete" permissions in the server
-    And user "Alice" has accepted the share "Shares/simple-folder" offered by user "Brian" in the server
-    And user "Alice" has logged in using the webUI
-    When the user opens folder "Shares" using the webUI
-    And the user shares folder "simple-folder" with user "Carol King" as "Custom permissions" with permissions "share, delete, update" using the webUI
-    Then the error message with header "Error while sharing." should be displayed on the webUI
-    And as "Carol" folder "Shares/simple-folder" should not exist in the server
-    And user "Carol" should not have received any shares in the server
```

Comment:
why deleting this test? is the resharer now allowed to increase the permissions or is the test not applicable at all?

Reply:
We deleted the test because the tested behavior (selecting an invalid role shows an error) changed completely. We want to write a new e2e test to check the new behavior (an invalid role isn't even showing) in the future.

Reply:
Ah cool, so deleting these tests is not a lost because they never passed and never would.
CC @ScharfViktor is this resharing behavior on your list to test in e2e or do we need a new issue to make sure we don't forget to write the e2e tests?

Reply:
I created issue, not to forget it. https://github.com/owncloud/web/issues/7041